notebooks/evaluation/EVALUATION_README.md

```diff
@@ -1,133 +1,86 @@
+The goal of this notebook is to create a filesystem of data ready-made for others to study:
 
+```text
+.
+└── evaluation/
+    └── data/
+        └── {version_number}/
+            ├── chunks           # the chunked documents of study
+            ├── raw              # the raw documents of study
+            ├── results          # results from using this data
+            ├── synthetic        # Q&A datasets of study created with RAGAS
+            └── embeddings/      # vector store dumps of pre-embedded document chunks
+                └── {model}
+```
 
+We use [RAGAS](https://ragas.io) to create synthetic data, but are more than happy for users to manually create datasets too.
 
-### Workflow for RAG hyperparameter experimentation
-1. Review the various locations in the codebase where `RAG prompts` are used
-2. Make a change in one or more of these locations
-3. Rebuild the core-api docker image (and any other images modified), using `docker compose rebuild --no-cache`
-4. Follow the rag_e2e_evaluation notebook to generate evaluation score for the modified Redbox RAG based on your changes
-5. Record your changes in **TBD**
+## 🔎 Studying a versioned dataset
 
+> [!NOTE]  
+> **Study** a versioned dataset with [`rag_e2e_evaluation.ipynb`](/notebooks/evaluation/rag_e2e_evaluation.ipynb)
 
-### RAG prompt locations
-#### 1. Prompts in core.py
-One prompt, the `_core_redbox_prompt` is located in [core.py](../../redbox/llm/prompts/core.py)
+The goal of this notebook is that everything you need to study a versioned dataset should be contained in a single place that evaluators can run end to end.
 
+We use [DeepEval](https://docs.confident-ai.com) to evaluate datasets.
 
-```python
-_core_redbox_prompt = """You are RedBox Copilot. An AI focused on helping UK Civil Servants, Political Advisors and\
-Ministers triage and summarise information from a wide variety of sources. You are impartial and\
-non-partisan. You are not a replacement for human judgement, but you can help humans\
-make more informed decisions. If you are asked a question you cannot answer based on your following instructions, you\
-should say so. Be concise and professional in your responses. Respond in markdown format.
+Right now the notebook only contains the final retrieval engine: the interplay of prompts and retriever. Chunking and embedding strategies will need to be loaded outside this notebook, though you can certainly assess them using it.
 
-=== RULES ===
+## ✅ Success! What now?
 
-All responses to Tasks **MUST** be translated into the user's preferred language.\
-This is so that the user can understand your responses.\
-"""
-```
+You've analysed a dataset, improved the RAG system, and you want those changes to make it into Redbox. Congratulations! 
```

Comment:
Do we want to add something about the margin of difference we want to see in the RAG evaluation metrics to be confident of an improvement?